python/cudf/cudf/_lib/io/utils.pyx

```diff
@@ -2,17 +2,20 @@
 
 from cpython.buffer cimport PyBUF_READ
 from cpython.memoryview cimport PyMemoryView_FromMemory
+from libcpp.map cimport map
 from libcpp.memory cimport unique_ptr
 from libcpp.vector cimport vector
 from libcpp.pair cimport pair
 from libcpp.string cimport string
 from cudf._lib.cpp.io.types cimport source_info, io_type, host_buffer
-from cudf._lib.cpp.io.types cimport sink_info, data_sink
+from cudf._lib.cpp.io.types cimport sink_info, data_sink, datasource
+from cudf._lib.io.kafka cimport kafka_consumer
 
 import codecs
 import errno
 import io
 import os
+import cudf
```

Comment:
import entire cudf?

Reply:
Yea, we use this pattern throughout the library to avoid circular references or expensive imports within functions. In general we'll likely want to address all of this at some point in the future, but for now this is par for the course.